### ISSUE TITLE
Update node-gyp dependency

### ISSUE THREAD

**Issue by rtrys** [instana/nodejs-sensor#312]
Can you help us to update `node-gyp` to latest version to have security updates?

```bash
@instana/collector@1.117.3
└─┬ @instana/autoprofile@1.117.3
  └── node-gyp@6.1.0 
```

Current version has a dependency with a vulnerability: 
`node-gyp@6.1.0 -> mkdirp@0.5.1 -> minimist@0.0.8`
https://www.sourceclear.com/vulnerability-database/security/prototype-pollution/javascript/sid-22678

**Comment by basti1302** [comment]
Yes! -> https://github.com/instana/nodejs-sensor/pull/311/files#diff-3d50435bd3bf001f82e54c4d746b17b897c1825373bdad2a319898aa15328fb6R35

You can expect a release with an updated dependency soon.

**Comment by basti1302** [comment]
Fixed with `@instana/collector@1.118.0`.

**Comment by rtrys** [comment]
awesome, thanks!
